node-commons/src/main/java/org/apache/iotdb/commons/partition/SchemaPartition.java

```diff
@@ -163,11 +173,61 @@ public void createSchemaPartition(
         .put(seriesPartitionSlot, regionReplicaSet);
   }
 
-  public void serialize(ByteBuffer buffer) {
-    // TODO: Serialize SchemaPartition
+  public void serialize(ByteBuffer buffer) throws IOException, TException {
+    try (ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream()) {
+      for (Entry<String, Map<TSeriesPartitionSlot, TRegionReplicaSet>> entry :
+          schemaPartitionMap.entrySet()) {
+        ReadWriteIOUtils.write(entry.getKey(), byteArrayOutputStream);
+        writeMap(entry.getValue(), byteArrayOutputStream);
+      }
+      byte[] toArray = byteArrayOutputStream.toByteArray();
+      buffer.putInt(toArray.length);
+      buffer.put(toArray);
+    }
   }
 
-  public void deserialize(ByteBuffer buffer) {
-    // TODO: Deserialize DataPartition
+  public void deserialize(ByteBuffer buffer) throws TException, IOException {
```

Comment:
Don't forget to add tests for deserialize~

Reply:
fixed